containers/nginx-storage/file-transfer-endpoints.conf

```diff
@@ -0,0 +1,77 @@
+
+    location /download/ {
+        limit_except        GET { deny all; }
+        auth_request        /download-auth/;
+        if ($request_uri ~ "/download/(.*)") {
+            set $download_url $1;
+        }
+        alias /srv/storage/;
+    }
+
+    location /download-auth/ {
+        internal;
+        proxy_pass                $gatekeeper_api_backend/gatekeeper/download-auth/$download_url;
+        proxy_pass_request_body   off;
+        proxy_set_header          Content-Length "";
+    }
+
+
+    location = /upload/ {
+        auth_request /upload-auth/;
+        auth_request_set $checksum $upstream_http_concent_file_checksum;
+        auth_request_set $size     $upstream_http_concent_file_size;
```

Comment:
Please align this.

Reply:
Done.

Reply:
Why is not correct now ? This is not change in this pull request.

Reply:
If you did not touch this line in this commit (only copied it) then it can stay. But please refactor it later.